core/src/main/java/bio/terra/pearl/core/config/DataStudyManagerConfig.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.pearl.core.config;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+@Component
+@Getter @Setter
+public class DataStudyManagerConfig {
```

Comment:
It's in config because originally I was trying to use `@ConfigurationProperties`, but that doesn't work when it's coming from `core`. I'll move it into `DataStudyManagerClient`.